File: Home2WorkAPI/src/main/java/it/gruppoinfor/home2workapi/model/Rank.java
package it.gruppoinfor.home2workapi.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Rank implements Serializable {
    @SerializedName("Shares")
    @Expose
    private Integer shares;
    @SerializedName("MonthShares")
    @Expose
    private Integer monthShares;
    @SerializedName("LastMonthShares")
    @Expose
    private Integer LastMonthShares;
    @SerializedName("SharedDistance")
    @Expose
    private Integer sharedDistance;
    @SerializedName("MonthSharedDistance")
    @Expose
    private Integer monthSharedDistance;
    @SerializedName("LastMonthSharedDistance")
    @Expose
    private Integer lastMonthSharedDistance;

    public Integer getShares() {
        return shares;
    }

    public void setShares(Integer shares) {
        this.shares = shares;
    }

    public Integer getMonthShares() {
        return monthShares;
    }

    public void setMonthShares(Integer monthShares) {
        this.monthShares = monthShares;
    }

    public Integer getLastMonthShares() {
        return LastMonthShares;
    }

    public void setLastMonthShares(Integer lastMonthShares) {
        LastMonthShares = lastMonthShares;
    }

    public Integer getSharedDistance() {
        return sharedDistance;
    }

    public void setSharedDistance(Integer sharedDistance) {
        this.sharedDistance = sharedDistance;
    }

    public Integer getMonthSharedDistance() {
        return monthSharedDistance;
    }

    public void setMonthSharedDistance(Integer monthSharedDistance) {
        this.monthSharedDistance = monthSharedDistance;
    }

    public Integer getLastMonthSharedDistance() {
        return lastMonthSharedDistance;
    }

    public void setLastMonthSharedDistance(Integer lastMonthSharedDistance) {
        this.lastMonthSharedDistance = lastMonthSharedDistance;
    }
}

File: Home2WorkAPI/src/main/java/it/gruppoinfor/home2workapi/model/Rank.kt
package it.gruppoinfor.home2workapi.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Rank : Serializable {
    @SerializedName("Shares")
    @Expose
    var shares: Int = 0
    @SerializedName("MonthShares")
    @Expose
    var monthShares: Int = 0
    @SerializedName("LastMonthShares")
    @Expose
    var lastMonthShares: Int = 0
    @SerializedName("SharedDistance")
    @Expose
    var sharedDistance: Int = 0
    @SerializedName("MonthSharedDistance")
    @Expose
    var monthSharedDistance: Int = 0
    @SerializedName("LastMonthSharedDistance")
    @Expose
    var lastMonthSharedDistance: Int = 0
}
